packages/cysync-core/src/constants/i18n/en.ts

```diff
@@ -1541,6 +1541,46 @@ Download from <a href="https://cypherock.com/get-started">https://cypherock.com/
 * No new updates to CySync v1 except security update
 `,
     },
+    editAccount: {
+      tab1: {
```

Comment:
Don't use generic names like `tab1`, `section3` etc. Instead try to add a meaningful name for the object.

```suggestion
      accountSelection: {
```